system/jlib/jtrace.hpp

```diff
@@ -164,11 +164,16 @@ class jlib_decl OwnedSpanScope
 public:
     OwnedSpanScope() = default;
     OwnedSpanScope(ISpan * _ptr);
+    OwnedSpanScope(const OwnedSpanScope& rhs) = delete;
+    OwnedSpanScope(OwnedSpanScope&& rhs) = default;
     ~OwnedSpanScope();
 
     inline ISpan * operator -> () const         { return span; }
     inline operator ISpan *() const             { return span; }
 
+    inline OwnedSpanScope& operator=(ISpan * ptr) { set(ptr); return *this; }
```

Comment:
I am not sure about this.  This is likely to cause leaks to be introduced if it is used in the form

ownedscope = createServerSpan()

You could use setown instead, but that is slightly error prone.  Probably safest to keep the other changes in this PR and delete this assignment operator.

You could explicitly enable 

   inline OwnedSpanScope& operator=(OwnedSpanScope&& ) = default;
   
to allow scope = std::move(OwnedSpanScope(create....));